apps/src/code-studio/pd/components/constants.jsx

```diff
@@ -0,0 +1,15 @@
+import {PropTypes} from 'react';
+
+const SchoolInfoPropType = PropTypes.shape({
```

Comment:
This appears to be only used in the `CustomSchoolInfo` component. Any reason it needs to be in its own file? And if it is, should we name the file something more specific than just `constants`?

Reply:
it's also used in `SchoolAutocompleteDropdownWithCustomFields`, but I am totally open to naming suggestions!

Reply:
schoolConstants.js?

Reply:
If it's just this prop type, I'd go with `SchoolInfoPropType.js` and export it as the default.